tests/test_loss.py

```diff
@@ -0,0 +1,31 @@
+import inspect
+import unittest
+
+import monai
+import torch
+from parameterized import parameterized
+from test_utils import TestCase, func_req_additional_args
+
+from trainlib.loss import get_loss
+
+monai_losses = [item[0] for item in inspect.getmembers(monai.losses) if item[0].endswith("Loss")]
+torch_losses = [item[0] for item in inspect.getmembers(torch.nn) if item[0].endswith("Loss")]
+monai_losses = [x for x in monai_losses if not func_req_additional_args(getattr(monai.losses, x))]
+torch_losses = [x for x in torch_losses if not func_req_additional_args(getattr(torch.nn, x))]
+
+LOSSES = monai_losses + torch_losses
+
+
+class TestLoss(TestCase):
+    @parameterized.expand(LOSSES)
+    def test_init(self, loss_name):
+        "Simple test if losses can be initialized"
+        config = self.config
```

Comment:
Does this copy or point to the reference?

Reply:
Not sure. It should copy... but probably points. Seems to work well regardless, as the same key is overwritten over and over again. But I'll explicitly copy the dict.